fix: WEB-207 SVG images now have the proper src, width and alignment

Zougouda opened the pull request.

Zougouda committed:
fix: WEB-207 SVG images now have the proper src, width and alignment

Zougouda commented:
Unit test still missing for now and coming soon ™️

Zougouda committed:
fix: WEB-207 now with associated Unit Test

PiiXiieeS committed:
Merge branch 'main' into bugfix/web-207

PiiXiieeS commented on src/proxy-page/proxy-page.service.ts:
```diff
@@ -101,6 +102,7 @@ export class ProxyPageService {
     } else if (type !== 'notitle') {
       addHeaderTitle()(this.context);
     }
+    fixSVG(this.config)(this.context)
```
We have a different configuration in lint or prettier, because all the lines have semicolon and yours not. 
Linter is showing an error in my local configuration for this line. We need to find the right setting to get the same controls and enforce semicolon, like it was initially.

PiiXiieeS replied:
I made some changes in the linter configuration so we should get now more control on the format.

PiiXiieeS committed:
build: add prettier conf in eslint

- added redundant prettier controls in eslintrc to ensure alignment
between workstations and developers
- .prettierrc is kept for backwards compatibility

PiiXiieeS committed:
Merge branch 'main' into bugfix/web-207

Zougouda committed:
Merge branch 'main' into bugfix/web-207

Zougouda committed:
Merge branch 'main' into bugfix/web-207
